infrastructure/src/test/java/org/corfudb/infrastructure/log/StreamLogChecksumTest.java

```diff
@@ -0,0 +1,23 @@
+package org.corfudb.infrastructure.log;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class StreamLogChecksumTest {
+
+    @Test
+    public void testChecksumBytes() {
+        assertEquals(506166820, StreamLogFiles.Checksum.getChecksum("corfuDb".getBytes()));
```

Comment:
Can you use assertj's assertThat and isEquals?

Reply:
I still don't understand why we can't use junit asserts?